orchestra/tests/test_task_lifecycle.py

```diff
@@ -498,7 +498,8 @@ def test_malformed_assignment_policy(self):
             slug='machine_step',
             is_human=False,
             assignment_policy={
-                'policy': 'previously_completed_steps',
+                'policy': ('orchestra.assignment_policies.'
+                           'simple.previously_completed_steps'),
                 'steps': ['step_0']
```

Comment:
what is this?

i'll add kwargs though, was going to add it in a later PR but I think it makes the most sense now